src/components/Cart/CartCount/CartCount.js

```diff
@@ -1,31 +1,38 @@
 import React from 'react';
 import { useState } from 'react';
+import { useEffect } from 'react';
+
 import * as S from './CartCountStyle';
 
-const CartCount = () => {
+const CartCount = ({ price, onCountChange, stock }) => {
   const [count, setCount] = useState(1);
 
-  const increaseCount = () => {
-    if (count >= 5) return;
-    setCount(prev => prev + 1);
-  };
-
-  const decreaseCount = () => {
-    if (count <= 1) return;
-    setCount(prev => prev - 1);
-  };
-
+  useEffect(() => {
+    onCountChange(count);
+  }, [count]);
   return (
     <S.CartCountInput>
-      <S.CartCountMinus onClick={decreaseCount}>-</S.CartCountMinus>
-      <S.CartCountButton>{count}</S.CartCountButton>
-      <S.CartCountPlus
-        onClick={() => {
-          increaseCount();
-        }}
-      >
-        +
-      </S.CartCountPlus>
+      <S.CartCountInputWrap>
+        <S.CartCountMinus
+          onClick={() => {
+            if (count <= 1) return;
+            setCount(prev => prev - 1);
+          }}
+        >
```

Comment:
```suggestion

const decreaseCount = () => {
            if (count <= 1) return;
            setCount(prev => prev - 1);
          }
return(
...
        <S.CartCountMinus
          onClick={decreaseCount}
        >
```
JSX에서 이벤트 함수는 분리 해 주세요